README.md

```diff
@@ -32,11 +32,11 @@
 - [📝 License](#license)
 
 
-# 📖 Hello Microverse <a name="about-project"></a>
+# 📖 Capstone1-project <a name="about-project"></a>
```

Comment:
- [ ] Kindly give your project a proper title. Instead of naming it Capstone-1, I would recommend naming it to whatever the project is related it. In your case, it is related to wine tasting. 
- [ ] Kindly remove this section from your README file. ⬇️
![image](https://user-images.githubusercontent.com/33268927/226114831-891622a7-dad1-4d93-b997-8ce3f79f3ad5.png)